.dotnet/src/Generated/OpenAIModelFactory.cs

```diff
@@ -40,6 +40,46 @@ public static TranscribedSegment TranscribedSegment(int id = default, long seekO
                 serializedAdditionalRawData: null);
         }
 
+        public static AssistantModificationOptions AssistantModificationOptions(string model = null, string name = null, string description = null, string instructions = null, IEnumerable<ToolDefinition> defaultTools = null, ToolResources toolResources = null, IDictionary<string, string> metadata = null, float? temperature = null, float? nucleusSamplingFactor = null, AssistantResponseFormat responseFormat = null)
```

Comment:
@joseharriaga - Was the generated code not up to date? Why these methods are generated after running the ` .scripts/Invoke-CodeGen.ps1` script?

Reply:
It's weird. I wonder if having `init` was preventing the generator from generating these. In any case, this model factory is currently `internal`, so don't worry about it.